lib/gruf/instrumentation/request_logging/hook.rb

```diff
@@ -139,6 +141,29 @@ def sanitize(params = {})
         def options
           super().fetch(:request_logging, {})
         end
+
+        def redact!(parts, i, params, redacted_string)
```

Comment:
Can you please add docblocks to each method with param/return descriptions and a summary?